addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -72,7 +72,7 @@ def _holiday_status_id_domain(self):
         'Number of Days', compute='_compute_from_holiday_status_id', store=True, readonly=False, tracking=True, default=1,
         help='Duration in days. Reference field to use when necessary.')
     number_of_days_display = fields.Float(
-        'Duration (days)', compute='_compute_number_of_days_display',
```

Comment:
Hello.

This is not respecting our stable policy.

Converting a computed field into a stored field on a stable release is forbidden, as this is a database schema modification.

See our stable policy for more information: https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean